src/components/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,21 @@
+.login-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: fit-content;
```

Comment:
Try and keep these sorted alphabetically as you go if you can

Reply:
Ill do that. However Ill also sort by section. Like width and height can be together since thats sizing. Ill also put all flex stuff together